cmd/pebble/cmd_help_test.go

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2022 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main_test
+
+import (
+	"github.com/jessevdk/go-flags"
+	. "gopkg.in/check.v1"
+
+	pebble "github.com/canonical/pebble/cmd/pebble"
+)
+
+func (s *PebbleSuite) TestHelpCommandWorks(c *C) {
+	rest, err := pebble.Parser(pebble.Client()).ParseArgs([]string{"help", "add"})
+
+	flagsErr, ok := err.(*flags.Error)
+	c.Assert(ok, Equals, true)
+	c.Assert(flagsErr.Type, Equals, flags.ErrHelp)
```

Comment:
Re `c.Assert` vs `c.Check`, I know it's a bit confusing, but I generally try to use `Assert` only when the test can't continue, so in this case we would use `Assert` for the `ok == true` check (because the following tests would fail/panic if you didn't), but then `c.Check` for the rest, because then the test failure will show all the check failures at once.